specs/signature-envelope-cose.md

```diff
@@ -239,6 +239,58 @@ The final signature envelope is a `COSE_Sign1_Tagged` object, consisting of Payl
 )
 ```
 
+## COSE Hash Envelope for signing arbitrary blobs
+Notary Project also supports [COSE Hash Envelope](https://datatracker.ietf.org/doc/draft-ietf-cose-hash-envelope/05/) when signing an arbitrary blob. COSE Hash Envelope is essentially another format of [COSE Sign1 signature structure](https://datatracker.ietf.org/doc/html/rfc8152#section-4.2). 
```

Comment:
We will only support one signature format for blob signing, which is COSE Hash Evenlop. So we only need to describe what is the payload and what does COSE hash evenlope look like, we don't need to do a comparision.